cerebrium/scaling/graceful-termination.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: "Graceful Termination"
```

Comment:
Also worth linking from the other places in our docs where we talk about pre-emption or handling sigterm. I think its in Scaling apps -> instance management